test/recipe-repo-test.js

```diff
@@ -0,0 +1,50 @@
+const chai = require('chai')
+const expect = chai.expect
+
+import sampleRecipeData from '../src/data/sampledata.js'
```

Comment:
Since we are planning to have sample data for ingredients, recipes and users we may want to consider making this more specific? Just a suggestion :)